server/src/main/java/org/opensearch/index/store/RemoteSegmentStoreDirectory.java

```diff
@@ -57,6 +74,11 @@ public final class RemoteSegmentStoreDirectory extends FilterDirectory implement
      */
     public static final String SEGMENT_NAME_UUID_SEPARATOR = "__";
 
+    /**
+     * Number of bytes in the segment file to store checksum
+     */
+    private static final int SEGMENT_CHECKSUM_BYTES = 8;
```

Comment:
We need this variable to calculate checksum of checksum for data integrity verification during segment uploads. Ref: https://github.com/opensearch-project/OpenSearch/pull/7119/files#diff-f57628264c235a133d2d6db458404e07a43f18026e40ed5e956536c03add9925R670

While matching checksums, we don't need this variable because:
1. During remote data integrity verification, we don't need to calculate another checksum post upload to compare the expectedChecksum with.
2. During local data integrity verification, we calculate the value of `actualChecksum` by using the checksum values from [`ResettableCheckedInputStreams`](https://github.com/opensearch-project/OpenSearch/blob/16c9d5e3b3ba8c0545303bff8e5af54db83ab09a/server/src/main/java/org/opensearch/common/blobstore/transfer/RemoteTransferContainer.java#L209).